doc/Style.md

```diff
@@ -4,6 +4,45 @@ This document describes the style used within Agora.
 It covers the style itself, which should be clear and unambiguous,
 as well as more design aspects on how we approach / solve problems.
 
+## Commits
+
+- A commit must be a self-contained unit of logical change.
+
+- A commit should not contain unrelated changes mixed into the same commit.
```

Comment:
Kinda redundant with point 1, no ?

Reply:
I'm just trying to make it more accessible to non-native speakers by driving the point.